tests/index.test.js

```diff
@@ -0,0 +1,387 @@
+        const faktorial = 5;
+        const expected = 120;
+
+        const actual = getFaktorial(faktorial);
+
+        assert.deepEqual (actual, expected)
+
+    }); 
+});     
+
+describe ('getSumNumber', () => {
+    it ('should return  sum number (12)', () => {
+        const number = 12;
+        const expected = 3;
+
+        const actual = getSumNumber (number)
+
+        assert.deepEqual (actual, expected)
+
+    })
+});
+
+describe ('revert (num) ', () => {
+    it ('should return revert (123)', () => {
+        const num = 123;
+        const expected = 321;
+
+        const actual = revert (num);
+
+        assert.deepEqual (actual, expected);
+    });
+});
+
+describe ('getMinofArray (numArray)', () => {
+    it ('should return get min of array [1,2,3,4]', () => {
+        const numArray = [1,2,3,4];
+        const expected = 1;
+
+        const actual = getMinOfArray (numArray);
+
+        assert.deepEqual (actual, expected);
+    });
+
+});
+     
+describe ('getMaxofArray (numArray)', () => {
+    it ('should return get max of array [1,2,3,4]', () => {
+        const numArray = [1,2,3,4];
+        const expected = 4;
+
+        const actual = getMaxOfArray (numArray);
+
+        assert.deepEqual (actual, expected);
+    });
+
+});
+
+describe ('indexOfMax (arr)', () => {
+    it('should return index of max array [1,2,3,4]', () => {
+        const index = [1,2,3,4];
+        const expected = 3;
+
+        const actual = indexOfMax (index)
+
+        assert.deepEqual (actual, expected)
+    });
+});
+
+describe ('indexOfMin (arr)', () => {
+    it('should return index of min array [1,2,3,4]', () => {
+        const index = [1,2,3,4];
+        const expected = 0 ;
+
+        const actual = indexOfMin (index);
+
+        assert.deepEqual (actual, expected);
+    });
+});
+
+describe ('sumOddIndex (arr)', () => {
+    it ('should return sum odd index [1,2,3,4,5,6]', () => {
+        const arr = [1,2,3,4,5,6];
+        const expected = 12;
+
+        const actual = sumOddIndex(arr);
+
+        assert.deepEqual(actual, expected);
+    });
+});
+
+describe ('selectionSort (arr)', () => {
+    it ('should return selection sort [2,5,1,3,7,4]', () => {
+        const arr = [2,5,1,3,7,4];
+        const expected = [1,2,3,4,5,7];
+
+        const actual = selectionSort (arr);
+
+        assert.deepEqual (actual, expected);
+    });
+});
+
+describe ('getNameDay (dayNumber)', () => {
+    it('should return get name day (1,2,3,4,5,6,7)', () => {
+        const dayNumber = 1;
+        const expected =  "понедельник";
+
+        const actual = getNameDay (dayNumber);
+
+        assert.deepEqual (actual, expected);
+    });
+    it('should return get name day (1,2,3,4,5,6,7)', () => {
+        const dayNumber = 2;
+        const expected =  "вторник";
+
+        const actual = getNameDay (dayNumber);
+
+        assert.deepEqual (actual, expected);
+    });
+    it('should return get name day (1,2,3,4,5,6,7)', () => {
```

Comment:
empty line before each `it`
tests should have different names. How will you know which of them failed?